src/Integration.Vsix/CFamily/PortedFromJava/MsvcDriver.cs

```diff
@@ -430,6 +454,24 @@ public static Request ToRequest(CFamilyHelper.Capture[] captures)
                     {
                         request.Flags |= Request.OperatorNames;
                     }
+                    if ((request.Flags & Request.CPlusPlus17) != 0 && !alignedNewDisabled)
+                    {
+                        request.Flags |= Request.AlignedNew;
+                    }
+
+                    var cppLatestImpliesUnpermissive = (major > 19) || (major == 19 && minor >= 28);
```

Comment:
Ported from https://github.com/SonarSource/sonar-cpp/blob/3dc9a31ba6a8d66e2e113c2d64290ef0e23cf9ae/sonar-cfamily-plugin/src/main/java/com/sonar/cpp/analyzer/MsvcDriver.java#L335

The original code uses a "version" class; I extracted the methods from there, keeping the original names